backend/src/main/java/gov/cdc/usds/simplereport/service/model/UserInfo.java

```diff
@@ -33,9 +34,11 @@ public UserInfo(ApiUser user, Optional<OrganizationRoles> orgwrapper, boolean is
         orgwrapper.map(OrganizationRoles::getGrantedRoles).orElse(Set.of()).stream()
             .collect(Collectors.toList());
     orgwrapper.map(OrganizationRoles::getGrantedPermissions).ifPresent(permissions::addAll);
-    this.facilities =
+    List<Facility> facilityList =
         orgwrapper.map(OrganizationRoles::getFacilities).orElse(Set.of()).stream()
             .collect(Collectors.toList());
+    facilityList.sort(Comparator.comparing(Facility::getFacilityName));
```

Comment:
Should this sort be case insensitive? 🤔 Looks like we can make use of this https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#CASE_INSENSITIVE_ORDER